client/core/core.go

```diff
@@ -3153,10 +3153,13 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 	if err != nil {
 		return nil, 0, err
 	}
+	c.notify(newWalletStateNote(fromWallet.state()))
+
 	err = prepareWallet(toWallet)
 	if err != nil {
 		return nil, 0, err
 	}
+	c.notify(newWalletStateNote(toWallet.state()))
```

Comment:
These address the issue, but what if we did this notify from `connectAndUnlock`, and then only if there is a wallet state change (i.e. it does either connect or unlock)? We could forgo wallet state notes when they convey no new info.

Reply:
Sounds good, will give it a try thanks.

Reply:
Done 👍🏽